LedApi/src/api.py

```diff
@@ -55,7 +56,14 @@ def theater_chase() -> Response:
 
 @app.route("/rainbow")
 def rainbow() -> Response:
-    led_request = LedRequest("rainbow")
+    led_request = LedRequest("rainbow", wait_ms = 20)
+    publish("leds", json.dumps(led_request.__dict__))
+    return Response(status=200)
+
+
+@app.route("/sleep")
+def rainbow() -> Response:
+    led_request = LedRequest("rainbow", wait_ms = 350)
     publish("leds", json.dumps(led_request.__dict__))
     return Response(status=200)
```

Comment:
You don't actually need to make a new function, you could just put the wait_ms into the JSON body and then use that. 

So only need to keep 'rainbow()'

Reply:
So when you send a POST to rainbow, you just add "wait_ms": 350 or "wait_ms": 50